tests/functional/i/import_outside_toplevel.rc

```diff
@@ -0,0 +1,2 @@
+[IMPORTS]
+allow-any-import-level=astroid
```

Comment:
I think this option is specific enough to be added in the same file.

Reply:
Will do!

Reply:
I meant it's ok there's nothing to change here 😄